src/components/AlteracaoDeCardapio/DRE/Relatorio/index.jsx

```diff
@@ -0,0 +1,293 @@
+import React, { Component } from "react";
+import HTTP_STATUS from "http-status-codes";
+import BaseButton, { ButtonStyle, ButtonType } from "../../../Shareable/button";
+import { Redirect } from "react-router-dom";
+import { reduxForm } from "redux-form";
+import { FluxoDeStatus } from "../../../Shareable/FluxoDeStatus";
+import { prazoDoPedidoMensagem, corDaMensagem } from "./helper";
+import { stringSeparadaPorVirgulas } from "../../../../helpers/utilities";
+import { ModalRecusarSolicitacao } from "../../../Shareable/ModalRecusarSolicitacao";
+import {
+  getAlteracaoCardapio,
+  DREConfirmaAlteracaoCardapio
+} from "../../../../services/alteracaoDecardapio.service";
+import { getDiasUteis } from "../../../../services/diasUteis.service";
+import { meusDados } from "../../../../services/perfil.service";
+import { dataParaUTC } from "../../../../helpers/utilities";
+import { toastSuccess, toastError } from "../../../Shareable/dialogs";
+import "../style.scss";
+import "./style.scss";
+
+class Relatorio extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      unifiedSolicitationList: [],
+      uuid: null,
+      meusDados: null,
+      redirect: false,
+      showModal: false,
+      ehInclusaoContinua: false,
+      alteracaoDecardapio: null,
+      prazoDoPedidoMensagem: null,
+      listaDeStatus: []
+    };
+    this.closeModal = this.closeModal.bind(this);
+  }
+
+  setRedirect() {
+    this.setState({
+      redirect: true
+    });
+  }
+
+  renderizarRedirecionamentoParaPedidos = () => {
+    if (this.state.redirect) {
+      return <Redirect to="/dre/alteracoes-de-cardapio" />;
+    }
+  };
+
+  componentDidMount() {
+    const urlParams = new URLSearchParams(window.location.search);
+    const uuid = urlParams.get("uuid");
+
+    meusDados().then(response => {
```

Comment:
meusDados().then(meusDados => {
      this.setState({
        meusDados
      });
    });